torch/csrc/generic/methods/SparseTensor.cwrap

```diff
@@ -72,24 +72,13 @@ PyObject * THSPTensor_(size)(PyObject *self, PyObject *args, PyObject *kwargs)
 
```

Comment:
General point: Can you call `contiguous()` before __repr__? It's weird that the printed tensor changes when you call a function for which this is a const input argument, e.g. mm.